pkg/sdkv2/resources/environment_acceptance_test.go

```diff
@@ -346,6 +347,108 @@ func TestAccDbtCloudEnvironmentResourceConnection(t *testing.T) {
 	})
 }
 
+// TestAccDbtCloudEnvironmentResourceProjectUpdate tests that environments are not incorrectly updated when
+// there is an update made to a project. Specifically tests the issue linked below where a project update
+// would cascade a connection_id to all environments in the project.
+// https://github.com/dbt-labs/terraform-provider-dbtcloud/issues/334
+func TestAccDbtCloudEnvironmentResourceProjectUpdate(t *testing.T) {
+	dbtVersionLatest := "latest"
+	environmentNameDev := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	environmentNameProd := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	projectName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	projectDescription := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	projectDescription2 := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: acctest_helper.TestAccProtoV6ProviderFactories,
+		CheckDestroy:             testAccCheckDbtCloudEnvironmentDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDbtCloudEnvironmentResourceDualConnectionConfig(
+					projectName,
+					projectDescription,
+					environmentNameDev,
+					environmentNameProd,
+					dbtVersionLatest,
+				),
+				ConfigStateChecks: []statecheck.StateCheck{},
```

Comment:
I think that this line is not needed, is it?

Reply:
Ah, yes, I'll remove that. I was experimenting with a different type of test check and accidentally left that in.